api/v1/_submissions/PKPBackendSubmissionsHandler.inc.php

```diff
@@ -134,6 +134,7 @@ public function getSubmissions($slimRequest, $response, $args) {
 					break;
 
 				case 'isIncomplete':
+				case 'isInactive':
```

Comment:
Instead of a `boolean`, let's use an `int` that represents the number of days since the last activity. Let's call it `daysInactive`. This way, the REST API can support more than one measure of inactivity, even if _for now_ the UI only supports one (30 days).